### PR TITLE
HDF1.14 compatibility

### PR DESCRIPTION
**Pull request by PieterTack**

### CODE REVIEW

**Comment by PieterTack** [comment]
The idea would also be to get rid of the gnu autotools here, and limit ourselves to meson.

File: python/polycap.pyx [hunk 1]
```diff
@@ -121,6 +121,17 @@ def __send_google_analytics_launch_event():
 
 threading.Thread(target=__send_google_analytics_launch_event).start()
 
+# Some code from h5py/_errors.pyx to retain HDF5 functionality across multiple versions
```
Comment (tschoonj) [inline python/polycap.pyx:124]: Copying that code from h5py here verbatim is not going to work, as it needs to be translated into C and add [here](https://github.com/PieterTack/polycap/blob/master/src/polycap-transmission-efficiencies.c#L81-L155).

**Comment by tschoonj** [comment]
> The idea would also be to get rid of the gnu autotools here, and limit ourselves to meson.

I would do that in a separate pull request, and I would do that first as it will allow you to simplify the CI config tremendously

File: python/polycap.pyx (hunk 1 above)
Comment (PieterTack) [reply python/polycap.pyx:124]: Yup, was reaching the same conclusion :D

**Comment by PieterTack** [comment]
> 

Alright, will do that first then :)
Git-wise, do you think it's best to then temporarily close this PR and focus on the other topic first, or doesn't it matter? I'm not very familiar with working with multiple PRs in git :)

**Comment by tschoonj** [comment]
Since there is no valuable code in this PR so far, easiest is to just close it and work on the CI changes in a new one

**Pull request closed without merging**